modules/lightning_features/lightning_api/modules/api_test/tests/src/Functional/ApiTestBase.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Drupal\Tests\api_test\Functional;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use Psr\Http\Message\ResponseInterface;
+
+abstract class ApiTestBase extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'lightning_headless';
```

Comment:
Lightning doesn't ship with the lightning_headless profile, so I'm not sure how this could work...?

Reply:
Not sure how this worked either. Switched this to Lightning since this module lives within it and won't be available in testing or minimal.